src/scripts/destinyTrackerApi/bulkFetcher.js

```diff
@@ -19,32 +19,41 @@ class bulkFetcher {
     };
   }
 
-  getBulkFetchPromise(stores) {
-    if (stores.stores.length === 0) {
+  _getBulkFetchPromise(stores) {
+    if (!stores.length) {
       return this.$q.resolve();
     }
 
-    var weaponList = this._itemListBuilder.getWeaponList(stores.stores, this._reviewDataCache);
+    const weaponList = this._itemListBuilder.getWeaponList(stores, this._reviewDataCache);
 
     if (!weaponList.length) {
       return this.$q.resolve();
     }
 
-    var promise = this.$q
-              .when(this.getBulkWeaponDataPromise(weaponList))
+    const promise = this.$q
+              .when(this._getBulkWeaponDataEndpointPost(weaponList))
               .then(this.$http)
               .then(this._trackerErrorHandler.handleErrors, this._trackerErrorHandler.handleErrors)
-              .then((response) => { return response.data; });
+              .then((response) => response.data);
 
     this._loadingTracker.addPromise(promise);
 
     return promise;
   }
 
-  bulkFetch(stores) {
-    this.getBulkFetchPromise(stores)
+  /**
+   * Fetch the DTR community scores for all weapon items found in the supplied stores.
+   *
+   * @param {any} storesContainer
```

Comment:
This doesn't really help me. It doesn't say anything about what `storesContainer` is or what properties it might have.